wave_vision/src/fast_detector.cpp

```diff
@@ -0,0 +1,87 @@
+// Libwave Headers
+#include "wave/vision/fast_detector.hpp"
+
+namespace wave {
+
+FASTDetector::FASTDetector(const std::string &config_path) {
+    // Extract parameters from .yaml file.
+    ConfigParser parser;
```

Comment:
No, it isn't. The detector could also be created by the user manually setting the configuration parameters. I decided to use a config file in order to maintain similarity with @Jebediah's scan matching interface.